source/Grid/Grid.test.js

```diff
@@ -1507,4 +1508,33 @@ describe('Grid', () => {
       expect(cellRendererCalls[1].style.width).toEqual(50)
     })
   })
+
+  it('should not pull from the style cache while scrolling if there is an offset adjustment', () => {
+    let cellRendererCalls = []
+    function cellRenderer (props) {
+      cellRendererCalls.push(props)
+    }
+
+    const grid = render(getMarkup({
+      cellRenderer,
+      width: 100,
+      height: 100,
+      rowHeight: 100,
+      columnWidth: 100,
+      rowCount: DEFAULT_MAX_SCROLL_SIZE * 2 / 100, // lots of offset
```

Comment:
Well, the height of the inner container needs to be over DEFAULT_MAX_SCROLL_SIZE to trigger this. Doing `DEFAULT_MAX_SCROLL_SIZE / rowHeight` will get us exactly enough rows to fill  the container -- so I figured I might as well double it to be sure :P

I'm happy to add comments explaining it, or if you have a better suggestion for the test use that.